### ISSUE TITLE
App does not allow for Dance Parties

### ISSUE THREAD

**Issue by bbugh** [dladowitz/fist-to-five#1]
Problem: I tried to put Dance Parties in as a subject but the app crashed.

Expectation: Dance Parties are an allowed subject.

**Comment by bbugh** [comment]
:dancer: :dancers: :dancer:

**Comment by dladowitz** [comment]
Weird, I specifically added Dance Parties as a validation. Should have allowed them. 

![screen shot 2015-03-31 at 5 12 19 pm](https://cloud.githubusercontent.com/assets/1074877/6932063/3b01b55e-d7c9-11e4-9adc-6bc172f7dd3e.png)

But seriously, did the app crash for you? Seems to be working for me. Not sure what's going on. 
No real logging setup.

**Comment by bbugh** [comment]
Yeah it crashed. Any time I tried to enter something it gave me a failure!

**Comment by dladowitz** [comment]
I'm guessing you are not signed in?
Looks like I have an issue with creating subjects while not being signed
in.
Try signing in.
Anonymous answers should be fine.

This is what happens when you decided not to test. I have no idea what
state anything is in. I knew shit like this would happen.

**Comment by dladowitz** [comment]
Ok, I just added some protections to for the subject creator to sign in. 
Should be working. 

Viva Testingggggggggggggggg!!!!!!!!
